workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfiguration.cs

```diff
@@ -56,6 +56,16 @@ internal List<string> Validate()
             {
                 errors.Add($"{GdkToolsConfigurationWindow.DevAuthTokenPathLabel} cannot be empty.");
             }
+            else
+            {
+                if (!DevAuthTokenDir.Contains("/Resources/")
+                    && !DevAuthTokenDir.Contains("Resources/")
```

Comment:
This should be `StartsWith`.

Imagine: `Assets/MyTotallyNotReallyResources/thing.txt`